examples/typescript/cdk8s-plus-elasticsearch-query/package.json

```diff
@@ -17,14 +17,14 @@
     "upgrade:next": "npm i cdk8s@next cdk8s-cli@next"
   },
   "dependencies": {
-    "cdk8s": "0.0.0",
-    "cdk8s-plus-17": "0.0.0",
+    "cdk8s": "1.0.0-beta.6",
```

Comment:
These are intentionally `0.0.0` so that we can align the versions on release (basically grep for that value and replace). (script [here](https://github.com/awslabs/cdk8s/blob/ffce8c696ce502ec4a14f65b715474faa7f74c7b/tools/align-version.sh)).

Is this causing a bug for you?